cfme/infrastructure/provider/kubevirt.py

```diff
@@ -0,0 +1,45 @@
+from cfme.utils.providers import get_crud
+from cfme.containers.provider.openshift import VirtualizationEndpoint
+from . import InfraProvider
+
+
+class KubeVirtProvider(InfraProvider):
+
+    def __init__(self, name=None, endpoints=None, key=None, provider_data=None,
+                 parent_provider=None, appliance=None):
```

Comment:
parent_provider isn't optional is it? The only way to get one of these providers in CFME is through adding a parent provider.

Reply:
Only specify parent_provider here, and use `**kwargs` to capture the rest of the args and pass them to `InfraProvider.__init__` so that the parent class's kwargs can change without breaking this.

Reply:
**Required**

parent_provider is a required kwarg.